lib/index.js

```diff
@@ -34,10 +35,10 @@ function getMetaData(command, root) {
   });
 }
 
-function getDependencies(command, root, targetFile, args) {
+function getDependencies(command, root, targetFile, allowMissing, args) {
```

Comment:
Would it help if you put this arg last, and set it to `false` by default? Backwards-compat consideration.

Reply:
`getDependencies()` is not an exported function, so backward-compat can be forgiven here I think. I feel it's better to put it before the more "generic" last argument which means "any extra args that will be concatenated the the command-line"